tools/flaky-test-reporter/main.go

```diff
@@ -55,12 +54,6 @@ func main() {
 	var repoDataAll []*RepoData
 	// Clean up local artifacts directory, this will be used later for artifacts uploads
 	err = os.RemoveAll(prow.GetLocalArtifactsDir()) // this function returns nil if path not found
-	if nil == err {
-		err = common.CreateDir(prow.GetLocalArtifactsDir())
-	}
-	if nil != err {
```

Comment:
this check is still needed

Reply:
I removed it here because I moved directory creation to the same place as artifact creation. Should I move it back?

Reply:
there is still an `err` above for cleaning up existing contents, which should be checked